content/en/blog/2024/elastic-contributes-continuous-profiling-agent.md

```diff
@@ -0,0 +1,152 @@
+overhead, low CPU (~1%), and memory usage in production environments.
+
+The Elastic profiling agent facilitates identifying non-optimal code paths,
+uncovering "unknown unknowns", and provides comprehensive visibility into the
+runtime behavior of all applications. Elastic’s continuous profiling agent
+provides support for a wide range of runtimes and languages, such as:
+
+- C/C++
+- Rust
+- Zig
+- Go
+- Java
+- Python
+- Ruby
+- PHP
+- Node.js
+- V8
+- Perl
+- .NET
+
+Additionally, organizations can meet sustainability objectives by minimizing
+computational wastage, ensuring seamless alignment with their strategic ESG
+goals.
+
+## Benefits to OpenTelemetry
+
+This contribution not only boosts the standardization of continuous profiling
+for observability but also accelerates its adoption as a key signal in
+OpenTelemetry. Customers benefit from a vendor-agnostic method of collecting
+profiling data correlating it with existing signals, like tracing, metrics, and
+logs, opening new potential for observability insights and a more efficient
+troubleshooting experience.
+
+### User Benefits of OpenTelemetry Profiling
+
+OpenTelemetry-based continuous profiling unlocks the following possibilities for
+users:
+
+- Improved customer experience: delivering consistent service quality and
+  performance through continuous profiling ensures customers have an application
+  that performs optimally, remains responsive, and is reliable.
+
+- Maximize gross margins: Businesses can optimize their cloud spend and improve
+  profitability by reducing the computational resources needed to run
+  applications. Whole system continuous profiling identifies the most expensive
+  functions (down to the lines of code) across diverse environments that may
+  span multiple cloud providers. In the cloud context, every CPU cycle saved
+  translates to money saved.
+
+- Minimize environmental impact: energy consumption associated with computing is
+  a growing concern (source:
+  [MIT Energy Initiative](https://energy.mit.edu/news/energy-efficient-computing/)).
+  More efficient code translates to lower energy consumption, contributing to a
+  reduction in carbon (CO2) footprint.
+
+- Accelerate engineering workflows: continuous profiling provides detailed
+  insights to help troubleshoot complex issues faster, guide development, and
+  improve overall code quality.
+
+- Improved vendor neutrality: a vendor-agnostic eBPF-based profiling agent
+  removes the need to use proprietary APM agents to collect profiling telemetry.
+
+With these benefits, SREs, devs, and DevOps, can now manage the overall
+application’s efficiency on the cloud while ensuring their engineering teams
+optimize it.
+
+Elastic remains deeply committed to the
+[OpenTelemetry vision and mission](https://opentelemetry.io/community/mission/)
+and will continue to contribute and participate in profiling and other areas of
+OpenTelemetry.
```

Comment:
We should close with some "joined statement" or an otel focused wording, since this is the otel not the elastic blog

Reply:
What do you suggest?

Reply:
added a proposal for a joint statement